### DISCUSSION TITLE
Project Maintainers

### DISCUSSION THREAD

**Discussion by robloo** [mono/SkiaSharp#1884] [General]
It seems @mattleibow is the sole maintainer of this repository. I'm not sure how scalable that is long term. For example, I am guessing he is on a well deserved vacation yet building with Uno Platform 4.x is completely broken right now. As several companies depend on SkiaSharp, having responsibility left to a single individual is a high risk.

I know this project is backed by Microsoft. However, I wonder who is on the SkiaSharp team besides @mattleibow? We need the ability to review patches and potentially do a preview release while he is away. This is just good stewardship of the project and helps mitigate risks for the vast number of consumers of this project.

You do great work @mattleibow and I know you are spread quite thin on time these days. I just want to start the discussion about other maintainers to this repo -- nothing against the amazing work you have done to date.

**Comment by Danielku15** [comment]
I have similar concerns. @mattleibow does an amazing job 🎉👍 . Nevertheless keeping up with the teams at Google is a tough job, hardly manageable by a single person. 

Prioritizing all the issues in the backlog is tough for one person. Even with the contributors helping out addressing them. In each release cycle the issues will compete with each other. The foundation doesn't get updated because we have critical issues on all the platform specific integration. Or vice versa: we need to make again a 2.80.x release because on Uno something broke, stealing precious (human) resources on the progress on an upgrade to m88 or m96.

Even if I contribute now a PR to move to m96 for the base library, we cannot or don't want to release it because there are platform things to be fixed too. 

The released version of SkiaSharp is 2.80 (based on m80) and the pre-release is working on 2.88 (based on m88). Chrome is already on m96. While I am not so much in the need of every new feature added (to have it exposed through SkiaSharp) I am in the need of having a consistent rendering with Chrome as I am comparing my rendered images in a visual regression suite across platforms. The fact that SkiaSharp is always quite behind to Chrome causes quite some maintenance overhead for me in my project. 

With all the bindings SkiaSharp supports, the project became quite monolithic and requires a very broad expertise. Maintaining feature parity with Skia and keeping it up to date is a totally different subject than maintaining the bindings for all platforms. 

I wonder what the best future strategy for this (amazing) project is. Both project/release-management wise and code/repository-management wise I think there are some considerations to make.

**Comment by Stamo-Gochev** [comment]
Any updates on this?
